week2/homework/src/index.js

```diff
@@ -1,3 +1,118 @@
 'use strict';
+let fs = require('fs');
+let order = process.argv[2];
+let orderDefinition = process.argv[3];
+let orderDefinition2 = process.argv[4];
 
-// TODO: Write the homework code in this file
+fs.readFile('./toDo.json', 'utf8', function (error, data) {
```

Comment:
Nesting everything inside a function is a bad practice.
It's difficult to understand what's going on and it will make testing these functions impossible.

Reply:
Now, all functions are stated to alone. Corrected